NavigationReactNative/src/ios/NVSceneController.m

```diff
@@ -1,15 +1,14 @@
 #import "NVSceneController.h"
-#import "NVSceneView.h"
 #import "NVNavigationBarView.h"
 #import "NVSearchBarView.h"
 
 @implementation NVSceneController
 {
-    NVSceneView *_view;
+    UIView *_view;
```

Comment:
You're working off an old version of the `NVSceneController`. Make sure you pull from master